examples/permissioned-token/test/failing-paths.js

```diff
@@ -0,0 +1,178 @@
+      fromAccount: bob.account, // asset manager account (fails otherwise)
+      appId: permissionsAppId,
+      payFlags: { totalFee: 1000 },
+      appArgs: ['str:change_permissions_manager'],
+      accounts: [elon.address]
+    };
+
+    // fails as fromAccount is not the current SSC manager
+    assert.throws(() => runtime.executeTx(txn), 'RUNTIME_ERR1009');
+  });
+
+  it('Force Transfer', () => {
+    // Opt-In to permissions SSC & Whitelist
+    whitelist(runtime, alice, alice.address, assetIndex, controllerAppID, permissionsAppId);
+    whitelist(runtime, alice, elon.address, assetIndex, controllerAppID, permissionsAppId);
+    whitelist(runtime, alice, bob.address, assetIndex, controllerAppID, permissionsAppId);
+    syncAccounts();
+
+    // Opt-In to ASA
+    optInToASA(runtime, bob.address, assetIndex);
+    optInToASA(runtime, elon.address, assetIndex);
+    syncAccounts();
+
+    // Issue some tokens to sender
+    issue(runtime, alice, bob, 150, controllerAppID, assetIndex, lsig);
+    syncAccounts();
+
+    // Fails as only manager can perform force transfer
+    assert.throws(() =>
+      forceTransfer(runtime, bob, elon, 20, assetIndex, controllerAppID, permissionsAppId, lsig, bob),
+    'RUNTIME_ERR1009'
+    );
+
+    // Fails as reciever will have more than 100 tokens post succesfull transaction
+    assert.throws(() =>
+      forceTransfer(runtime, bob, elon, 101, assetIndex, controllerAppID, permissionsAppId, lsig, alice),
+    'RUNTIME_ERR1009'
+    );
+  });
+
+  it('Token Transfer', () => {
+    fund(runtime, master, elon.address);
+    fund(runtime, master, bob.address);
+    fund(runtime, master, alice.address);
+    fund(runtime, master, lsig.address());
+    syncAccounts();
+    const amount = 20;
+
+    // Issue some tokens to Bob and Elon
+    optInToASA(runtime, elon.address, assetIndex);
+    optInToASA(runtime, bob.address, assetIndex);
+    syncAccounts();
+    issue(runtime, alice, elon, 50, controllerAppID, assetIndex, lsig);
+    issue(runtime, alice, bob, 50, controllerAppID, assetIndex, lsig);
+    syncAccounts();
+
+    // Cannot transfer directly
+    assert.throws(() => runtime.executeTx({
+      type: types.TransactionType.TransferAsset,
+      sign: types.SignType.SecretKey,
+      fromAccount: bob.account,
+      toAccountAddr: elon.address,
+      amount: amount,
+      assetID: assetIndex,
+      payFlags: {}
+    }), 'RUNTIME_ERR1505');
+
+    // Cannot transfer before being whitelisted
+    assert.throws(() =>
+      transfer(runtime, bob, elon, amount, assetIndex, controllerAppID, permissionsAppId, lsig),
+    'RUNTIME_ERR1009'
+    );
+
+    // Both sender and receiver needs to be whitelisted before transferring tokens
+    whitelist(runtime, alice, bob.address, assetIndex, controllerAppID, permissionsAppId);
+    assert.throws(() =>
+      transfer(runtime, bob, elon, amount, assetIndex, controllerAppID, permissionsAppId, lsig),
+    'RUNTIME_ERR1009'
+    );
+  });
```

Comment:
- Please also add a failing test - regarding receiver balance (> 100).
- We should also add a test where we don't include call to contoller, or call to permissions (both should fail). Maybe change sender of clawback & controller (it should also fail)

Reply:
> Please also add a failing test - regarding receiver balance (> 100).
I have added this. 

I don't think the second test is required, but I have added that too.